sdk/communication/AzureCommunicationCalling/CHANGELOG.md

```diff
@@ -1,5 +1,28 @@
 # Release History
 
+## 2.0.0-beta.1 (2021-07-14)
+### New features
+- `Recording` and `Transcription` features are decoupled from `Call` object and now can only be used via extensions.
+  For e.g. 
+```
+            let recordingFeature = self.call!.api(RecordingFeature.self)
+            recordingFeature.delegate = self.callObserver
+```
+
+### Breaking changes
+- `didChangeRecordingState` and `didChangeTranscriptionState` are moved out of `CallDelegate` into `RecordingFeatureDelegate` and into `TranscriptionFeatureDelegate`.
```

Comment:
Per updated guidance:
Should be prefaced with `Changed:`

Reply:
Did not understand ,where Changed should be ?

Reply:
- Changed: `didChangeRecordingState` and...

Reply:
ok.